contracts/Rentals.sol

```diff
@@ -97,6 +109,7 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver {
         address _operator,
         uint256 _rentalDays,
         uint256 _pricePerDay,
+        bytes _signature,
```

Comment:
I would move `_signature` to the end. It is going to be cheaper

Reply:
349944 → 349931 True :)